cmd/kubectl-testkube/commands/tests/deleteall.go

```diff
@@ -0,0 +1,29 @@
+package tests
+
+import (
+	"github.com/kubeshop/testkube/cmd/kubectl-testkube/commands/common"
+	"github.com/kubeshop/testkube/cmd/kubectl-testkube/commands/common/validator"
+	"github.com/kubeshop/testkube/pkg/ui"
+	"github.com/spf13/cobra"
+)
+
+func NewDeleteTestsCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "delete <testName>",
+		Short: "Delete tests",
+		Long:  `Delete tests by name`,
+		Args:  validator.TestName,
```

Comment:
Should be changed for all tests, we don't need name then

Reply:
good catch! fixed